rollup.config.js

```diff
@@ -101,16 +93,15 @@ export default [
 			}
 		]
 	},
-	/* compiler.js */
+	// Compiler: Generated CJS/UMD build for the compiler
```

Comment:
Do we still need the CJS version now that we're using Vitest and most things are ESM? Plus aren't the tests doing their own bundling step, so why bundle twice? If we do need it still, perhaps we could add a comment explaining why?

Reply:
ESlint, prettier and the REPL need it

Reply:
Don't they use the IIFE version that gets created during `PUBLISH`? I'm talking about the version that's being created here when `!PUBLISH`

Reply:
oh, actually I see below you've decided to get rid of the CJS version as suggested here: https://github.com/sveltejs/svelte/pull/8613#issuecomment-1557363470